### DISCUSSION TITLE
I am a student

### DISCUSSION THREAD

**Discussion by Ilkinhuseynov2** [community/community#61068] [GitHub Education]
### Select Topic Area

Question

### Body

Hello, I had two accounts on GitHub and I'm a student. Since I'm a student, I used my student email to create an account and got the GitHub Student Pack. Then, I deleted that account to transfer the benefits to my other primary account by adding the email to it. However, I haven't received the benefits on my new account yet. Could you please help me with this?

**Comment by Akash1134** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @Ilkinhuseynov2  We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. 

Also, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category [GitHub Education](https://github.com/orgs/community/discussions/categories/github-education) for you. Good luck!
